subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -627,8 +628,9 @@ static int __wifi_args_to_params(const struct shell *sh, size_t argc, char *argv
 	params->eap_ver = 1;
 	params->ignore_broadcast_ssid = 0;
 	params->bandwidth = WIFI_FREQ_BANDWIDTH_20MHZ;
+	params->verify_peer_cert = 0;
```

Comment:
should we default use `params->verify_peer_cert = 1` to keep the same logic before change?